src/mmw/js/src/data_catalog/templates/resultDetailsCuahsi.html

```diff
@@ -0,0 +1,79 @@
+<div class="result">
+    <div class="result-detail-header">
+        <button type="button" class="close" aria-label="Close">
+            <span aria-hidden="true">×</span>
+        </button>
+        <h2>
+            Site: {{ location }} {{ title }}
+        </h2>
+        <p>
+            Source: {{ service_org }} {{ service_title }}
+            <a data-toggle="popover" tabindex="0"
+               data-html="true" data-container="body" role="button"
+               data-content="{{ description }}"
+               data-template="<div class='popover'><div class='pull-right' id='popover-close-button'><i class='fa fa-times' /></div><div class='popover-content'></div><div class='arrow'></div></div>">
+                <i class="fa fa-info-circle black"></i>
+            </a>
+        </p>
+        <p>
+            <!-- TODO Make This Work -->
+            <a class="zoom" href="#">
+                <i class="fa fa-search-plus"></i> Zoom to extent
```

Comment:
I thought that all WDC results were point-based. If that's the case, it may not make sense to have a "zoom to extent" feature.